server/historian/packages/historian/config.json

```diff
@@ -28,5 +28,6 @@
     },
     "restGitService": {
         "disableGitCache": false
-    }
+    },
+    "storageUrl": ""
```

Comment:
see other comments, but I think `storageUrlOverride` would be better

Reply:
Is the plan for us to eventually only use the Historian config `storageUrl` (meaning we still fallback to the tenant provided url just while we transition to the new setup), or will we always do that? If it's just temporary, it might make sense to leave it just as `storageUrl` (since there will be no tenant-defined storage to override later)?

Reply:
We will eventually only use the Historian config storageUrl, instead of using the tenant-provided URL. So it should fine to leave it just as storageUrl.

Reply:
I see thank you for explaining. I was unaware that we intended to completely remove per-tenant url configuration from R11s.